libres/lib/include/ert/enkf/enkf.hpp

```diff
@@ -1,28 +0,0 @@
-/*
-   Copyright (C) 2011  Equinor ASA, Norway.
-
-   The file 'enkf.h' is part of ERT - Ensemble based Reservoir Tool.
-
-   ERT is free software: you can redistribute it and/or modify
-   it under the terms of the GNU General Public License as published by
-   the Free Software Foundation, either version 3 of the License, or
-   (at your option) any later version.
-
-   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
-   WARRANTY; without even the implied warranty of MERCHANTABILITY or
-   FITNESS FOR A PARTICULAR PURPOSE.
-
-   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
-   for more details.
-*/
-
-#define SPLASH_LENGTH 8
-const char *splash_text[SPLASH_LENGTH] = {
-    "                  8888888888          888    d8P  8888888888",
-    "                  888                 888   d8P   888       ",
-    "                  888                 888  d8P    888       ",
-    "                  8888888    88888b.  888d88K     8888888   ",
-    "                  888        888 '88b 8888888b    888       ",
-    "                  888        888  888 888  Y88b   888       ",
-    "                  888        888  888 888   Y88b  888       ",
-    "                  8888888888 888  888 888    Y88b 888       "};
```

Comment:
How can we delete this? Also, same comment, preferably different PR, or at least different commit.